packages/flutter_tools/lib/src/build_system/targets/localizations.dart

```diff
@@ -36,6 +36,7 @@ Future<void> generateLocalizations({
     'bin',
     'gen_l10n.dart',
   );
+
   final ProcessResult result = await processManager.run(<String>[
```

Comment:
There is one small missing feature: you could check that generate is true and if not warn that the sources won't be importable.

Reply:
I just [added the feature](https://github.com/flutter/flutter/pull/62395/files#diff-d27c3ed6f283118c1a9719f24ec8a4f5R42) and [a test](https://github.com/flutter/flutter/pull/62395/files#diff-e3ab382c96a655e902c9ff5ba96fa840R71). PTAL